### ISSUE TITLE
Wrong link for cmt_xs.pth

### ISSUE THREAD

**Issue by leondgarse** [ggjy/CMT.pytorch#1]
Thanks for sharing these!
I think your released `cmt_xs.pth` is actually a duplication of `cmt_small.pth`. Would it be convenient uploading a new one?
Btw, I've also ran some training earlier with a little different structure: [keras_cv_attention_models/cmt](https://github.com/leondgarse/keras_cv_attention_models/tree/main/keras_cv_attention_models/cmt). The main difference is that I'm using individual relative positional embedding for each attention block like [beit](https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/beit.py#L97). With 305 epochs, `cmt_t + 160 input` accuracy is `78.94`, and I think it's not bad considering the parameters and FLOPs.

**Comment by ggjy** [comment]
Thanks for pointing out that! I have uploaded the corrected ```cmt_xs.pth``` version. The result of individual relative positional embedding is great, and I also include your codebase in my readme.md.

**Comment by leondgarse** [comment]
Great! I've uploaded the ported `CMT_XS` h5 model in my repo too.
